docker/proxy/proxyctl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+case "$1" in
+  start)
+        docker start proxy
+        ;;
+  stop)
+        docker stop proxy
+        ;;
+  shell)
+        docker exec -it proxy /bin/bash
+        ;;
+  update)
+        docker stop proxy
+        docker container rm proxy
+
+        docker pull ipac/proxy
+        docker run -d \
+                   -p 80:80 \
+                   -p 443:443 \
```

Comment:
it you has more parameters to the docker containers the this file will have to deal with them